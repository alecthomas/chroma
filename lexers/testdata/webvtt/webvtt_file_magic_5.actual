WEBVTT without two line terminators following