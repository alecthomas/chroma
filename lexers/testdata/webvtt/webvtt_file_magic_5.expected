[
  {"type":"Error","value":"WEBVTT without two line terminators following"}
]
